### DISCUSSION TITLE
Airdrop

### DISCUSSION THREAD

**Discussion by Mugii450** [IBAX-io/go-ibax#4] [Q&A] [answered]
I've finished the project, what's the next step?

**Comment by HalloKyle** [comment, marked as answer]
The airdrop will last for 2 month please stay tuned！

**Reply by Roni48** [reply]
Saya butuh tempat untuk alamat BEP

**Reply by Roni48** [reply]
0xF44613ff3079dC9a2077033DcF42d69d32C79701

**Reply by Sinyorlaz** [reply]
what should i do to get the prize here

**Reply by Sinyorlaz** [reply]
> Airdrop 2 ay sürecek, lütfen bizi izlemeye devam edin！

what should i do to get the prize here

**Reply by HalloKyle** [reply]
Guyz please be patient, we are still in the testnet phase and we will launch our main network as soon as possible after the airdrop (around March or April). We will then open up the redemption of IBAX points to IBXC coins.

**Reply by Sinyorlaz** [reply]
Is wallet address required here for 5k ibax reward?

**Reply by HalloKyle** [reply]
nah  wallet address is free to use for everyone we will add that reward until the Mainnet is up no worry

**Reply by Sinyorlaz** [reply]
> Guyz lütfen sabırlı olun, hala testnet aşamasındayız ve ana ağımızı airdroptan sonra mümkün olan en kısa sürede başlatacağız (Mart veya Nisan civarında). Daha sonra IBAX puanlarının itfasını IBXC madeni paralarına açacağız.

I would be honored to work with you to help Turkish users.

**Reply by Sinyorlaz** [reply]
> KötüGötKyle

İbax wallet adress 1053-6537-9030-1519-5531 🤗

**Reply by HalloKyle** [reply]
I love Turkish， could you contact me on telegram on Monday，we are on weekends （you can find me （Kyle）on our telegram ）

**Reply by Sinyorlaz** [reply]
> I love Turkish， could you contact me on telegram on Monday，we are on weekends （you can find me （Kyle）on our telegram ）

Ok . See you later 🤗

**Comment by sempurna99** [comment]
information about the wallet whether we have to link our tesnet wallet sir on our web aidrop,.. 
because the menu to add a wallet on the web already exists.. 
Please help

**Reply by HalloKyle** [reply]
You can either link your testnet wallet address to your airdrop account by using your Twitter account or wait for the Mainnet to go live. 

Even after Mainnet goes live, the test network is not shut down (for other eco-developers as a test purpose), it shares the same algorithm as the main network, so when the main network is up and running, airdrop coins can be distributed to your testnet address properly and error-free, the principle behind this is like a cold wallet. 

After the mainnet goes live and we start distributing airdrops, a record of coins received by your wallet will be packaged and uploaded to the mainnet of the IBAX blockchain.

I hope I have answered your question.

You can check this video for how to bing a wallet to your airdrop account.

https://www.youtube.com/watch?v=kOWd-uIGwa4

**Reply by sempurna99** [reply]
Okay sir

**Comment by brsky-38** [comment]
Can you send your roadmap when will the coin be listed?

**Reply by HalloKyle** [reply]
Right now, we are in the test network phase until February 1st.

After that, we will have more or less 3 weeks to prepare the main network for launch, fix the bugs in the test network, upgrade the blockchain system, deploy servers and other preparations that take time.

If all goes well, we will launch the mainnet in March/April and start redeeming IBXC coins immediately!

In the meantime we will list our coins after the Mainnet is launched.

Right now we are contact exchanges if you got connections that can help us please contact me on telegram. (@IBAXKyle)

**Comment by Baro1905** [comment]
https://github.com/Baro1905/codeql-javascript-unsafe-jquery-plugin/pull/4/commits/b72aa8602bf1ba8e1cadb3ecdda8f246ede08049

**Reply by Baro1905** [reply]
[ethhub-e3301b90893a3cc91825747e69c17d335eee2f9c-1.zip](https://github.com/IBAX-io/go-ibax/files/9170461/ethhub-e3301b90893a3cc91825747e69c17d335eee2f9c-1.zip)

**Comment by Baro1905** [comment]
I hope IBAX start tranding

**Comment by MrDino3813** [comment]
Is there a recent airdrop on IBAX??
